SwiftSampleProject/SwiftyVokoderTests/SwiftyVokoderTests.swift

```diff
@@ -12,8 +12,39 @@ import Vokoder
 
 let grandMilwaukeeIdentifier = 30096
 
+struct CTAData {
+
+    static func allStopDictionaries() -> [[String: AnyObject]] {
```

Comment:
I started doing this in another project and decided to remove it. Seeing `JSONObject` everywhere was just confusing because at a glance, it looked like it was supposed to be a class or struct somewhere.